Update grafana and lambda ui documentation (#2200)
kyma-project/kyma: resources/monitoring/charts/grafana/README.md

Users can create their own **Grafana Dashboard** by using the Grafana UI as the dashboards are persisted even after the Pod restarts.

1. Create or modify a dashboard using Grafana UI.
2. Save the dashboard with a new name.

## Lambda dashboard

The lambda dashboard provides visualisation for specific lambda function metrics such as memory usage, CPU usage or success rate response.

You can access the dashboard directly from the lambda UI.

> ## Unique dashboard identifier

Reviewer: ```suggestion
## Unique Dashboard Identifier
```

> 1. Create or Modify a dashboard using Grafana UI

Reviewer: Could you also change this one to:
`Create or modify a dashboard using Grafana UI.` ? 

It's not changed within this task, but for consistency we can change it :)

2. Save the dashboard with a new name.

> ## Lambda Dashboard

Reviewer: ```suggestion
## Lambda dashboard
```

> The Lambda Dashboard provides visualisation for specific lambda function metrics such as Memory usage, CPU usage and success rate response, etc.

Reviewer: ```suggestion
The lambda dashboard provides visualisation for specific lambda function metrics such as memory usage, CPU usage or success rate response.
```

> This can be accessed from lambda ui directly.

Reviewer: ```suggestion
You can access the dashboard directly from the lambda UI.
```

> ## Unique Dashboard Identifier

Reviewer: ```suggestion
## Unique dashboard identifier
```

> The Unique Dashboard Identifier or uid allows having consistent URL’s for accessing dashboards from lambda ui. 

Reviewer: ```suggestion
The Unique Dashboard Identifier or UID allows having consistent URLs for accessing dashboards from the lambda UI. 
```

> This uid is defined in the `*-dashboard.json`

Reviewer: ```suggestion
This UID is defined in the `*-dashboard.json` file.
```

> >**Important:** Change the uid will break the url used to access specific dashboards from lambda ui.

Reviewer: ```suggestion
>**Note:** Changing the UID breaks the URL used to access specific dashboards from the lambda UI.
```
